CHANGELOG.md

```diff
@@ -15,6 +15,10 @@
 - Fix hero tile being next to a left or right aligned image @jackahl
 - Fix toolbar elements showing depending on user permissions @sneridagh
 
+### Internal
```

Comment:
@jackahl Changelog occurs in the latest version, do not get confused by the 4.0.0, your changelog goes there. Please, can you amend that?

Reply:
@sneridagh done